src/ci/docker/host-x86_64/mingw-check/check-default-config-profiles.sh

```diff
@@ -8,5 +8,6 @@ config_dir="../src/bootstrap/defaults"
 # Loop through each configuration file in the directory
 for config_file in "$config_dir"/*.toml;
 do
-    python3 ../x.py check --config $config_file --dry-run
+    # Disable CI mode, because it is not compatible with all profiles
+    python3 ../x.py check --config $config_file --dry-run --ci=false
 done
```

Comment:
Which config causing incompatibility? Using both `--ci=true` and `--ci=false` for dry-run and bootstrap tests would provide great coverage.

Reply:
`library` and `tools`, because they use `true` for `download-ci-llvm`. I don't think that we need "CI mode" coverage of all the profiles, because except for `dist` they are not designed to run on CI anyway. And `dist` is used on CI for all jobs, so if it didn't work we would fine about that very quickly :)